src/yaml/wmp/essentials.yaml

```diff
@@ -0,0 +1,42 @@
+group: wmp
+name: essentials
+version: "1.0.1"
+subfolder: 100-props-textures
+info:
+  summary: WMP Essentials
+  author: DocRorlach, WMP team, xxdita, Tyberius06
+  website: https://community.simtropolis.com/files/file/33620-wmp-essentials-v101/
+  images:
+    - https://www.simtropolis.com/objects/screens/monthly_2020_05/5ecbdd9244bb2_WMPEssentials.jpg.dd3230029af5130cfe134bb921ad791f.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2020_05/5ecbdd94de456_WMPEssentials01.jpg.df0e06940c1a04c18670ef3b7278b547.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2020_05/5ecbdd9685da1_WMPEssentials02.jpg.6b0cd032a598b827cbc172090d896016.jpg
+assets:
+  - assetId: wmp-essentials-v101
+    include:
+      - /WMP/
+
+variants:
+  - variant: { driveside: right }
+  - variant: { driveside: left }
+    dependencies:
+      - wmp:essentials-lhd
```

Comment:
No need for a separate LHD dependency package. Using
```yaml
- variant: { driveside: left }
  assets:
    - assetId: wmp-essentials-v101
      include:
        - /z___zWMP_LHD_Paths.dat
```
here should be fine, as `z__…` loads after `WMP Essentials.dat` when both files are installed in the same folder.